src/components/TextInput/TextInput.scss

```
@@ -0,0 +1,112 @@
+@use '~@quartz/styles/scss/borders';
+@use '~@quartz/styles/scss/color-scheme';
+@use '~@quartz/styles/scss/fonts';
+@use '~@quartz/styles/scss/helpers/resets';
+@use '~@quartz/styles/scss/tokens';
+
+.container {
+	position: relative;
+	border: 0;
+	padding: 0;
+	margin: 0;
+}
+
+.label {
+	@include fonts.maison-extended-700-1;
+
+	text-transform: uppercase;
+	color: color-scheme.$typography;
+	display: block;
+	line-height: 1.25;
+
+	&.required {
+		display: flex;
+		flex-wrap: wrap;
+
+		&::after {
+			content: '*';
+
+			@include fonts.maison-800-3;
+
+			color: color-scheme.$accent;
+			padding: 0 4px;
+			line-height: 0.8;
+		}
+	}
+}
+
+.input {
+	@include resets.text-input;
+	@include fonts.maison-500-3;
+
+	position: relative;
+	display: inline-block;
+	height: 44px;
+	width: 100%;
+	border-radius: tokens.$size-border-radius;
+	border: borders.$solid-interactive;
+	box-sizing: border-box;
+	font-size: 16px;
+	font-weight: 300;
+	color: color-scheme.$typography;
```

would inherit work here?
```suggestion
	color: inherit;
```